code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -328,25 +309,26 @@
 	dismantle_cooldown = 0
 
 //argument is so world start can clear out any turf differences quickly.
-/datum/excited_group/proc/self_breakdown(space_is_all_consuming = 0)
+/datum/excited_group/proc/self_breakdown(space_is_all_consuming = FALSE)
 	var/datum/gas_mixture/A = new
 
 	//make local for sanic speed
 	var/list/A_gases = A.gases
 	var/list/turf_list = src.turf_list
 	var/turflen = turf_list.len
-	var/space_in_group = 0
+	var/space_in_group = FALSE
 
 	for(var/t in turf_list)
 		var/turf/open/T = t
 		if (space_is_all_consuming && !space_in_group && istype(T.air, /datum/gas_mixture/immutable/space))
-			space_in_group = 1
+			space_in_group = TRUE
 			qdel(A)
 			A = new/datum/gas_mixture/immutable/space()
+			A_gases = A.gases //update the cache
```

Comment:
technically this doesn't break anything but it should be a minor speedup to not loop through the out of date cache

Reply:
speaking of breaking, maybe i should `break` the loop here...